docs/source/examples/learning-depolarizing-noise.myst

```diff
@@ -0,0 +1,221 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name:  python3
+---
+
+# Learning quasiprobability representations with a depolarizing noise model
+
+In this example, we demonstrate the workflow of learning quasiprobability representations of a `CNOT` gate with a depolarizing noise model,
+from Clifford circuit data.
+The depolarizing noise model is parameterized by the noise strength, `epsilon`.
+The resulting quasiprobability representations of the `CNOT` gate are then used to obtain an error-mitigated expectation value with Mitiq's
+{ref}`probabilistic error cancellation module <guide/pec/pec>`. 
+For a more in-depth description of the learning function used in this example, see the section on {func}`.learn_depolarizing_noise_parameter`
+in the API-doc.
+The learning-based PEC workflow was inspired by the procedure described in *Strikis et al. PRX Quantum (2021)* {cite}`Strikis_2021_PRXQuantum`.
+
+```{code-cell} ipython3
+import numpy as np
+from matplotlib import pyplot as plt
+from cirq import (
+    CXPowGate,
+    DepolarizingChannel,
+    Rx,
+    Rz,
+    I,
+    X,
+    Y,
+    Z,
+    LineQubit,
+    Circuit,
+    ops,
+    unitary,
+    InsertStrategy,
+)
+from mitiq import Executor, Observable, PauliString
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+from mitiq.cdr import generate_training_circuits
+from mitiq.cdr._testing import random_x_z_cnot_circuit
+from mitiq.pec import execute_with_pec
+from mitiq.pec.representations.depolarizing import (
+    represent_operation_with_local_depolarizing_noise,
+)
+from mitiq.pec.representations.learning import (
+    biased_noise_loss_function,
+    learn_depolarizing_noise_parameter,
+)
+```
+
+Since the learning-based workflow uses the function {func}`.cdr.generate_training_circuits` from Mitiq's {ref}`Clifford data regression module
+<guide/cdr/cdr>` to generate the near-Clifford training circuits, the input circuit must be one that is compiled into the Rx-Rz-CNOT gateset. 
+Here we use a simple Rx-Rz-CNOT circuit, with an (optional) seed for reproducibility.
+
+```{code-cell} ipython3
+circuit = random_x_z_cnot_circuit(
+    LineQubit.range(2), n_moments=5, random_state=np.random.RandomState(1)
+)
+print(circuit)
```

Comment:
Can we show an example with more than 1 CNOT gate?
You can try to change seed or `n_moments`, or just manually define the circuit without calling the CDR function. 

More than 1 CNOT allows us to test the technique one step further in the direction of a real use case.

If for some reason it doesn't work, we come back to the single CNOT case.

Reply:
Locally this works and takes about 5 min to run. We'll see what it does in CI :)